scripts/pr_review_test.py

```
@@ -2,461 +2,331 @@
-
-        _, kwargs = post.call_args
-        self.assertEqual(kwargs["timeout"], pr_review.DEFAULT_LLM_TIMEOUT_SECONDS)
-        self.assertEqual(
-            kwargs["json"]["max_completion_tokens"],
-            pr_review.DEFAULT_MAX_COMPLETION_TOKENS,
-        )
-        self.assertEqual(
-            kwargs["json"]["reasoning_effort"],
-            pr_review.FAST_REASONING_EFFORT,
-        )
+        with self.assertRaisesRegex(pr_review.ModelOutputError, "schema"):
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)